DB-schema-migration.md

```diff
@@ -6,6 +6,8 @@ This documentation includes information about possible strategies regarding DB s
 
 For RMB based modules a DB migration will be done automatically based on the content of the schema.json provided within the module. Indeed there are exceptions, but each such case must be considered separately.
 
+DB Schema migration script can only support forward compatibility. To rollback the DB, please check Database rollback commands topic https://docs.liquibase.com/commands/home.html and https://docs.liquibase.com/tools-integrations/maven/commands/maven-rollback.html?Highlight=backward
+
```

Comment:
Thanks you for improving the migration documentation.
RMB doesn't use Liquibase, therefore I would not mention this.
I propose a different wording mentioning a workaround: https://github.com/folio-org/raml-module-builder/pull/987/files

Reply:
Thank you for correctly phrasing it. Sounds good to me.